simtools/applications/simulate_prod.py

```diff
@@ -140,25 +139,22 @@ def _parse(description=None):
     )
     config.parser.add_argument(
         "--from_azimuth_direction",
-        help="Direction from which the primary reaches the atmosphere",
-        type=str.lower,
+        help=(
+            "Direction from which the primary reaches the atmosphere."
```

Comment:
`reaches the atmosphere` is really confusing me. Would it be different if the Earth would have no atmosphere? (we ignore now other facts like breathing difficulties). I guess this is the direction the particle is traveling / shower axis is pointing towards. This would be clearer.

Reply:
You made me realise that it's actually wrong, it's the pointing direction, not the `from_azimuth_direction`. We do the conversion to CORSIKA `PHI` later. I suggest to instead change the option to a simple `azimuth_angle` and change the docstring to be `Telescope pointing direction in azimuth`, OK?

Reply:
OK!